tests/testthat/test-lint_file.R

```diff
@@ -119,6 +119,7 @@ test_that("lint() results from file or text should be consistent", {
     "x<-1",
     "x+1"
   )
+  on.exit(unlink(file))
```

Comment:
I didn't use `on.exit()` here because I want to ensure that lintr does not assume that the file exists when linting with both `filename` and `text` supplied.

Reply:
ah ok, please revert